assets/src/edit-story/components/panels/videoAccessibility.js

```diff
@@ -41,9 +41,21 @@ function VideoAccessibilityPanel({ selectedElements, pushUpdate }) {
     'resource',
     DEFAULT_RESOURCE
   );
-  const poster = getCommonValue(selectedElements, 'poster') || resource.poster;
-  const title = getCommonValue(selectedElements, 'title') || resource.title;
-  const alt = getCommonValue(selectedElements, 'alt') || resource.alt;
+
+  const _defaultForUndefined = (s, d) => (s === undefined ? d : s);
```

Comment:
No need for underscore. Also, if it's completely pure - let's move out of the function body on the top level.

Reply:
I've removed the method as I can just use the 3rd param of getCommonValue.